deploy.sh

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p jq -i bash
+
+# shellcheck shell=bash
+
+set -euo pipefail
+
+
+# For first deployment to bare server
+if [[ ${1:-} == '--prime' ]]; then
+    shift
+
+    BARE_SERVER=1
```

Comment:
What? Why not? I use all caps all the time for script-global variables.

Reply:
Because shell is just another programming language and there is absolutely no reason not to follow the standard convention that variables are camelCase and constants are ALL_CAPS.

But if you are looking for a more authoritative argument, then: we don’t have an official style guide, so we should use Google’s, which is more-or-less de-facto standard in the industry: https://google.github.io/styleguide/shellguide.html#s7.2-variable-names.